### PR TITLE
add issue templates for bugs, features, questions

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
[TOOL-1463](https://internal.pingcap.net/jira/browse/TOOL-1463)

### What is changed and how it works?
Add issue templates for bugs, features, questions to help contributors to propose an issue.
The effect of this PR is shown at [here](https://github.com/lichunzhu/tidb-binlog/issues/new/choose).

### Check List 

Tests

Code changes

Side effects

Related changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug Report"
+about: Something isn't working as expected
+---
```
Comment (lichunzhu) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Actually we don't have a label for bug here. Should we add one?

File: .github/ISSUE_TEMPLATE/general-question.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+## General Question
+
+Before asking a question, make sure you have:
+
+- Searched existing Stack Overflow questions.
+- Googled your question.
+- Searched open and closed [GitHub issues](https://github.com/pingcap/tidb-binlog/issues?utf8=%E2%9C%93&q=is%3Aissue)
+- Read related documentation:
```
Comment (july2993) [inline .github/ISSUE_TEMPLATE/general-question.md]: just show the official  doc links

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (july2993) [reply .github/ISSUE_TEMPLATE/bug-report.md]: you can add one

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Bug Report
+
+Please answer these questions before submitting your issue. Thanks!
+
+1. Which component are you using?
+(arbiter/binlogctl/drainer/pump/reparo)
```
Comment (july2993) [inline .github/ISSUE_TEMPLATE/bug-report.md]: use (pump/drainer/...) ?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+3. What did you expect to see?
+
+
+
+4. What did you see instead?
+
+
+
+5. What version of component are you using?
+(run `component -V` in terminal, component can either be arbiter/binlogctl/drainer/pump/reparo)
```
Comment (july2993) [inline .github/ISSUE_TEMPLATE/bug-report.md]: drop `component can ....` 
provide the relate downstream type and version of drainer

File: .github/ISSUE_TEMPLATE/general-question.md (hunk 2 above)
Comment (lichunzhu) [reply .github/ISSUE_TEMPLATE/general-question.md]: Revised.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 3 above)
Comment (lichunzhu) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Q1 has been deleted.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 4 above)
Comment (lichunzhu) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Revised.

**Comment by lichunzhu** [comment]
@july2993 Revised, PTAL.

File: .github/ISSUE_TEMPLATE/general-question.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  - [TiDB-Binlog Official Tutorial](https://pingcap.com/docs-cn/dev/how-to/get-started/tidb-binlog/)
```
Comment (july2993) [inline .github/ISSUE_TEMPLATE/general-question.md]: just use https://pingcap.com/docs/

**Pull request merged**